operator/v1alpha1/register.go

```diff
@@ -10,7 +9,7 @@ import (
 var (
 	GroupName     = "operator.openshift.io"
 	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1alpha1"}
-	schemeBuilder = runtime.NewSchemeBuilder(addKnownTypes, configv1.Install)
```

Comment:
The scheme is here because one scheme embeds types from another.  I think it's probably a mistake to include this group in the registration for `oc` here https://github.com/openshift/origin/blob/master/cmd/oc/oc.go#L54

Reply:
ok, i'll create a PR to remove that in origin.